pkg/object/httpserver/mux.go

```diff
@@ -261,6 +261,31 @@ func (mp *MuxPath) matchPath(ctx context.HTTPContext) bool {
 	return false
 }
 
+func (mp *MuxPath) handleRewrite(ctx context.HTTPContext) {
+	if mp.rewriteTarget == "" {
+		return
+	}
+
+	path := ctx.Request().Path()
+
+	if mp.path != "" {
+		ctx.Request().SetPath(mp.rewriteTarget)
+		return
+	}
+
+	if mp.pathPrefix != "" {
+		path = strings.Replace(path, mp.pathPrefix, mp.rewriteTarget, 1)
+		ctx.Request().SetPath(path)
+		return
+	}
+
+	if mp.pathRE != nil {
+		path = mp.pathRE.ReplaceAllString(path, mp.rewriteTarget)
+		ctx.Request().SetPath(path)
+		return
+	}
```

Comment:
This is incorrect, for example:
if both `mp.path` and `mp.pathPrefix` are not empty and the request matches `mp.pathPrefix`, the path should be rewritten at L277, but was actually rewritten at L272, which is wrong.

Reply:
I missed the point that path and prefix and regexp can co-exist, I previously thought they were a mutually exclusive relationship, I'll fix that